test/e2e/common/util_cleanup.go

```diff
@@ -321,10 +322,38 @@ func ForceDeleteMayastorPods() (bool, int, error) {
 // "Big" sweep, attempts to remove artefacts left over in the cluster
 // that would prevent future successful test runs.
 func CleanUp() bool {
-	podCount, delPodsErr := DeleteAllPods()
-	pvcCount, delPvcErr := DeleteAllPvcs()
-	pvCount, delPvErr := DeleteAllPvs()
-	msvCount, delMsvErr := DeleteAllMsvs()
+	var errs []error
```

Comment:
Maybe explain what the bool return value means.

Reply:
Done